src/domain.py

```diff
@@ -85,6 +85,6 @@ def build_juju_unit_status(pod_status):
     elif pod_status.is_running and not pod_status.is_ready:
         unit_status = MaintenanceStatus("Pod is getting ready")
     elif pod_status.is_running and pod_status.is_ready:
-        unit_status = ActiveStatus()
+        unit_status = ActiveStatus("Unit is ready")
```

Comment:
There was a discussion somewhere in the operator framework github or in the irc channel where it was suggested not to actually add a message with `ActiveStatus` since the unit's status can actually change anytime after this point and the charm won't always be informed in time and, thus, unable to set the message appropriately in time too which can be confusing for an operations team.

Reply:
Okay, makes sense. Removed.